x/feegrant/keeper/keeper.go

```diff
@@ -71,13 +71,13 @@ func (k Keeper) GrantFeeAllowance(ctx sdk.Context, granter, grantee sdk.AccAddre
 	return nil
 }
 
-// RevokeFeeAllowance removes an existing grant
-func (k Keeper) RevokeFeeAllowance(ctx sdk.Context, granter, grantee sdk.AccAddress) error {
+// revokeFeeAllowance removes an existing grant
+func (k Keeper) revokeFeeAllowance(ctx sdk.Context, granter, grantee sdk.AccAddress) error {
 	store := ctx.KVStore(k.storeKey)
 	key := types.FeeAllowanceKey(granter, grantee)
-	_, found := k.GetFeeGrant(ctx, granter, grantee)
-	if !found {
-		return sdkerrors.Wrap(sdkerrors.ErrUnauthorized, "fee-grant not found")
+	_, err := k.getFeeGrant(ctx, granter, grantee)
+	if err != nil {
+		return err
 	}
```

Comment:
Can we check this before creating `store`, `key`?

Reply:
Yeah, make sense.